sample templates/end_year/SchoolReportTemplate_SchoolYear__ZeugnisTemplate_Schuljahr.tex

```diff
@@ -0,0 +1,84 @@
+\input{SchoolReportTemplate_ZeugnisTemplate_Packages.tex}
+
+\pagenumbering{gobble}
+\setlength\parindent{0pt}
+\setlength\cellspacetoplimit{2pt}
+\setlength\cellspacebottomlimit{2pt}
+
+\newcommand{\tableGradeWidth}{.050\linewidth}
+\newcommand{\tableGradeWidthLevel}{.060\linewidth}
+\newcommand{\tableLineBreak}{\vspace{.2em}\newline}
```

Comment:
commands should be part of the `...Commands.tex`

Reply:
... like it was changed on the main branch in March f08d451d5628601933f2c31a14f18cd01c8e6fc8

Reply:
fixed